specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,537 @@
+- Proposal Name: Multi-Token-Standard
+- Start Date: 2021/07/24
+- Issue(s): #245.
+# Summary
+[summary]: #summary
+
+A standard interface for  a multi token standard that supports fungible, semi-fungible, and tokens of any type, allowing for ownership, transfer, and batch transfer of tokens generally regardless of specific type.
+
+# Motivation
+[motivation]: #motivation
+Having a single contract represent both NFTs and FTs can greatly improve efficiency as demonstrated by Enjin Coin. The ability to make batch requests with multiple asset classes can reduce a many transactions transaciton to a single transaction to trade around both NFTs and FTs that are a part of same token contract.
+
+Having this will also increase Near's ability to work interoperably with other chains. This will reduce the complexity required to represent these emerging asset classes.
+
+
+Prior art:
+- EIP-1155 : https://github.com/ethereum/EIPs/issues/1155
+
+- This NEP derives some examples and format from: https://github.com/near/NEPs/pull/21
+
+- Example of an NFT Series: https://github.com/near-apps/gnr8
```

Comment:
```suggestion
```

I think we'll have to remove this for now as it does not abide by the NFT standard at this time. An example of this being off: https://github.com/near-apps/gnr8/blob/6ae5916b4c21056e91ab89beff9aa8a56cefd02b/contracts/nft/src/nft_core.rs#L28